Misc/NEWS.d/next/Core and Builtins/2019-01-02-15-23-28.bpo-35636.HezsOG.rst

```diff
@@ -0,0 +1 @@
+remove redundant check in unicode_hash()
```

Comment:
This change does not need a news entry. It does not change the behavior, and the affect on performance is insignificant.

Reply:
ok, news entry removed.